### PR TITLE
Added meis2 expressing GABAergic cell

### LINKED ISSUE #966: [NTR] Meis2 cortical GABAergic
Note - please check that the term does not already exist (check OLS: https://www.ebi.ac.uk/ols/ontologies/cl)

**Preferred term label:**
Meis2 expressing GABAergic cortical neuron (Mus musculus) 

**Synonyms**

**Definition (free text, please give PubMed ID in format PMID:XXXXXX)**
A GABAergic type(? - using terminology from reference) located in the cerebral cortex that expresses the gene Meis2 in mouse
PMID:33372656

**Parent cell type term (check the hierarchy here https://www.ebi.ac.uk/ols/ontologies/cl):**
GABAergic Interneuron? (see below)

**What is the anatomical structure that the cell is a part of? Please check Uberon: https://www.ebi.ac.uk/ols/ontologies/uberon**
Cerebral Cortex

**Your nano-attribution (ORCID)**

**Any additional notes or concerns**
Not 100% sure about neuronal nature of this given that in Fig 10 of https://www.biorxiv.org/content/10.1101/2020.10.19.343129v1 it is in the "other" box. However, [PMID:33372656 states it as a GABAergic
](https://elifesciences.org/download/aHR0cHM6Ly9jZG4uZWxpZmVzY2llbmNlcy5vcmcvYXJ0aWNsZXMvNTk5MjgvZWxpZmUtNTk5Mjgtc3VwcDItdjIueGxzeA--/elife-59928-supp2-v2.xlsx?_hash=FiWSXqPj3SGTXyKbv%2B4i012DkRMUAtra6Bc99Zn8oN8%3D)

### PR DESCRIPTION
**Pull request by shawntanzk**
Fixes #966

### CODE REVIEW

**Comment by shawntanzk** [comment]
see issue #966 for discussion

File: src/ontology/cl-edit.owl [hunk 1]
```diff
@@ -27678,6 +27674,17 @@ SubClassOf(obo:CL_4023036 ObjectSomeValuesFrom(obo:RO_0002100 obo:UBERON_0000956
 SubClassOf(obo:CL_4023036 ObjectSomeValuesFrom(obo:RO_0002215 obo:GO_0061534))
 SubClassOf(obo:CL_4023036 ObjectSomeValuesFrom(obo:RO_0002292 obo:PR_P32848))
 
+# Class: obo:CL_4023043 (meis2 expressing cortical GABAergic cell (Mus musculus))
+
+AnnotationAssertion(Annotation(oboInOwl:hasDbXref "PMID:33372656") obo:IAO_0000115 obo:CL_4023043 "A GABAergic type located in the cerebral cortex that expresses the gene Meis2 in mouse")
+AnnotationAssertion(<http://purl.org/dc/elements/1.1/creator> obo:CL_4023043 <http://orcid.org/0000-0001-7258-9596>)
+AnnotationAssertion(oboInOwl:SubsetProperty obo:CL_4023043 <http://purl.obolibrary.org/obo/cl#BDS_subset>)
+AnnotationAssertion(rdfs:label obo:CL_4023043 "meis2 expressing cortical GABAergic cell (Mus musculus)")
+SubClassOf(obo:CL_4023043 obo:CL_0000617)
```
Comment (shawntanzk) [inline src/ontology/cl-edit.owl:27683]: Current asserting it under GABAergic neuron

File: src/ontology/cl-edit.owl [hunk 2]
```diff
@@ -27678,6 +27674,17 @@ SubClassOf(obo:CL_4023036 ObjectSomeValuesFrom(obo:RO_0002100 obo:UBERON_0000956
...
+SubClassOf(obo:CL_4023043 ObjectSomeValuesFrom(obo:RO_0002100 obo:UBERON_0000956))
+SubClassOf(obo:CL_4023043 ObjectSomeValuesFrom(obo:RO_0002162 obo:NCBITaxon_10090))
+SubClassOf(obo:CL_4023043 ObjectSomeValuesFrom(obo:RO_0002292 <http://www.informatics.jax.org/marker/MGI:108564>))
```
Comment (shawntanzk) [inline src/ontology/cl-edit.owl:27686]: not 100% sure why it didn't use PR:P97367 which I pulled in, but it is correct.

File: src/ontology/cl-edit.owl [hunk 3]
```diff
@@ -27689,6 +27685,17 @@ SubClassOf(obo:CL_4023036 ObjectSomeValuesFrom(obo:RO_0002162 obo:NCBITaxon_1009
 SubClassOf(obo:CL_4023036 ObjectSomeValuesFrom(obo:RO_0002215 obo:GO_0061534))
 SubClassOf(obo:CL_4023036 ObjectSomeValuesFrom(obo:RO_0002292 obo:PR_P32848))
 
+# Class: obo:CL_4023043 (meis2 expressing cortical GABAergic cell (Mus musculus))
+
+AnnotationAssertion(Annotation(oboInOwl:hasDbXref "PMID:33372656") obo:IAO_0000115 obo:CL_4023043 "A GABAergic type located in the cerebral cortex that expresses the gene Meis2 in mouse")
```
Comment (patrick-lloyd-ray) [inline src/ontology/cl-edit.owl:27690]: Recommend change to "A GABAergic cell located in the cerebral cortex that expresses meis2 in mouse."

Reading very carefully, the placement of 'in mouse' creates a slight ambiguity, but that's quibbling.

File: src/ontology/cl-edit.owl [hunk 4]
```diff
@@ -27509,6 +27493,18 @@ SubClassOf(obo:CL_4023013 obo:CL_0000679)
 SubClassOf(obo:CL_4023013 ObjectSomeValuesFrom(obo:RO_0002100 obo:UBERON_0000956))
 SubClassOf(obo:CL_4023013 ObjectSomeValuesFrom(obo:RO_0013001 obo:UBERON_0010225))
 
+# Class: obo:CL_4023014 (L5 Vip expressing cortical GABAergic interneuron (Mus musculus))
+
+AnnotationAssertion(Annotation(oboInOwl:hasDbXref "PMID:26420784") Annotation(oboInOwl:hasDbXref "PMID:26612957") obo:IAO_0000115 obo:CL_4023014 "A Vip GABAergic cortical interneuron found in L5. L5 Vip cells have mostly local morphology with some deep-projecting axons. They show only moderate resistance, comparable to that of Sst subclass and unlike typical Vip subclass cells that tend to show high input resistance. L5 Vip cells show particularly low resting membrane potential.")
```
Comment (patrick-lloyd-ray) [inline src/ontology/cl-edit.owl:27498]: If the other defs. include 'in mouse', then this should as well. I am unfamiliar with the standard so cannot settle the issue.

**Review by patrick-lloyd-ray — COMMENTED** [review]
Couple of in-line comments.

File: src/ontology/cl-edit.owl (hunk 4 above)
Comment (shawntanzk) [reply src/ontology/cl-edit.owl:27498]: You do make a good point - that entry is just reshuffled, but we should probably add in Mouse to all that are mouse specific. Will write a separate ticket on this.

**Comment by dosumis** [comment]
"meis2 in mouse" sounds like only expresses meis2 in mice.

**Comment by patrick-lloyd-ray** [comment]
> "meis2 in mouse" sounds like only expresses meis2 in mice.

This was the ambiguity problem above. Should 'in mouse' be included in the def or just moved to the gloss for cases like this?

**Comment by shawntanzk** [comment]
I'll change it to: 
"A mouse GABAergic cell located in the cerebral cortex that expresses meis2" 
should be clear that way

File: src/ontology/cl-edit.owl [hunk 5]
```diff
@@ -27697,6 +27718,144 @@ SubClassOf(obo:CL_4023043 ObjectSomeValuesFrom(obo:RO_0002100 obo:UBERON_0000956
...
+AnnotationAssertion(<http://purl.org/dc/elements/1.1/creator> obo:CL_4023038 <http://orcid.org/0000-0001-7258-9596>)
+AnnotationAssertion(oboInOwl:hasDbXref obo:CL_4023038 "ILX:0770163")
+AnnotationAssertion(oboInOwl:inSubset obo:CL_4023038 <http://purl.obolibrary.org/obo/cl#BDS_subset>)
+AnnotationAssertion(rdfs:label obo:CL_4023038 "L6b glutamatergic cortical neuron")
+SubClassOf(obo:CL_4023038 obo:CL_0000679)
+SubClassOf(obo:CL_4023038 ObjectSomeValuesFrom(obo:RO_0002100 obo:UBERON_0000956))
+
+# Class: obo:CL_4023040 (L2/3-6 intratelencephalic projecting glutamatergic cortical neuron)
+
+AnnotationAssertion(Annotation(oboInOwl:hasDbXref "DOI:10.1101/2020.10.19.343129") obo:IAO_0000115 obo:CL_4023040 "A intratelencephalic-projecting glutamatergic neuron with a soma found in cortical layers L2/3-6")
```
Comment (patrick-lloyd-ray) [inline src/ontology/cl-edit.owl]: Missing full stop, but otherwise everything looks good to me.

**Review by patrick-lloyd-ray — COMMENTED** [review]
Minor punctuation.

**Comment by shawntanzk** [comment]
> Minor punctuation.

oh that's outdated weird pullover from messy serialising, not in the final file change.

**Comment by shawntanzk** [comment]
@dosumis got a bit messy cause of a clashing IRI - let me unwind this first

**Comment by shawntanzk** [comment]
not 100% sure what happened with the file change that so many files are changed, but i've checked my github history and it seems that only the correct files are changed

**Comment by shawntanzk** [comment]
Weird changes - @shawntanzk to redo
